wordpress-mobile/gutenberg-mobile
# Move Gutenberg WebView related files to rn-gutenberg-bridge folder

Fixes parts of #2358

Discussion: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2063#discussion_r419769386

~WIP: Needs Android side changes.~

cc @ceyhun @cameronvoell 

To test iOS:
- Run the example app.
- Scroll to the last block of the example content.
- Press the (?) icon.
- Edit block in web browser.
- Check that gutenberg loads with the block.

PR submission checklist:
